### ISSUE TITLE
Module not found: Error: Package path ./lib/model/declarations is not exported from package

### ISSUE THREAD

**Issue by pjalinski** [lagoshny/ngx-hateoas-client#43]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂
```diff
diff --git a/node_modules/@lagoshny/ngx-hateoas-client/package.json b/node_modules/@lagoshny/ngx-hateoas-client/package.json
index 34b9785..481f116 100644
--- a/node_modules/@lagoshny/ngx-hateoas-client/package.json
+++ b/node_modules/@lagoshny/ngx-hateoas-client/package.json
@@ -53,7 +53,8 @@
       "es2020": "./fesm2020/lagoshny-ngx-hateoas-client.mjs",
       "es2015": "./fesm2015/lagoshny-ngx-hateoas-client.mjs",
       "node": "./fesm2015/lagoshny-ngx-hateoas-client.mjs",
-      "default": "./fesm2020/lagoshny-ngx-hateoas-client.mjs"
+      "default": "./fesm2020/lagoshny-ngx-hateoas-client.mjs",
+      "declarations": "./lib/modal/declarations.d.ts"
     }
   },
   "sideEffects": false
```

**Comment by lagoshny** [comment]
Hi @pjalinski ,

Some values exported from `declarations.d.ts`. What do you use that not exported?

**Comment by pjalinski** [comment]
GetOption,
  PagedGetOption,
  RequestOption,
  RequestParam,

Im building project using https://github.com/maximegris/angular-electron as template

**Comment by lagoshny** [comment]
@pjalinski ,
 Fixed in new versions for old Angular 2.3.9 and for the last Angular 3.0.9.
